app/infrastructure/exceptions/base_exception.py

```
@@ -0,0 +1,17 @@
+from typing import Optional
+
+from app.infrastructure.exceptions.error_type import ErrorType
+
+
+class InfrastructureBaseException(Exception):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)